KnowSheet/scripts/check-headers.sh

```diff
@@ -10,9 +10,16 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -DBRICKS_CHECK_HEADERS_MODE"
 LDFLAGS="-pthread"
 
+if [ $(uname) = "Darwin" ] ; then
+  CPPFLAGS+=" -stdlib=libc++ -x objective-c++ -fobjc-arc"
```

Comment:
@deathbaba , we are avoiding using `thread_local` as necessary.

Just printing a `#warning` in case it can be supported on Apple, but is not. We figured with Max that it would be the safest route for now.